NuRadioMC/simulation/simulation.py

```diff
@@ -1417,8 +1425,12 @@ def get_distance_cut(shower_energy):
                                                     self._propagator.get_number_of_raytracing_solutions(),
                                                     particle_mode=particle_mode)
 
-        efieldToVoltageConverter.begin()
-        channelGenericNoiseAdder.begin(seed=self._config['seed'])
+
+        # Only needed for data-driven noise
+        scale_dir = os.path.join(os.path.dirname(__file__), "../examples/RNO_G_trigger_simulation/data_driven_noise_files")
```

Comment:
egh... does this have to be in simulation.py?

Reply:
We can probably make it configureable - I can do that.